app/src/main/res/layout/content_main.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    app:layout_behavior="@string/appbar_scrolling_view_behavior"
+    tools:context="com.example.lhi06.residue.MainActivity"
+    tools:showIn="@layout/activity_main"
+    android:weightSum="1">
+
+    <EditText
+        android:id="@+id/request"
+        android:layout_width="0dp"
+        android:layout_height="59dp"
+        android:hint="@string/search_box"
+        app:layout_constraintRight_toLeftOf="@+id/searchButton"
+        tools:layout_constraintRight_creator="1"
+        app:layout_constraintBaseline_toBaselineOf="@+id/searchButton"
+        tools:layout_constraintBaseline_creator="1"
+        tools:layout_constraintLeft_creator="1"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintHorizontal_bias="1.0" />
+
+    <Button
+        android:id="@+id/searchButton"
+        android:layout_width="0dp"
```

Comment:
We always save these in the dimens file so we can easily reuse them again. That applies to all the hard coded dp